packages/terra-date-picker/package.json

```diff
@@ -29,7 +29,8 @@
     "moment": "^2.21.0",
     "popper.js": "^1.14.3",
     "prop-types": "^15.5.8",
-    "react-datepicker": "^0.64.0",
+    "react-onclickoutside": "^6.6.3",
+    "react-popper": "^0.7.4",
```

Comment:
Are we able to depend on 1.x.x of react-popper to at least match [react-datepicker's dependency version](https://github.com/Hacker0x01/react-datepicker/blob/v1.8.0/package.json#L96)? And are we able to get rid of the dependency on popper.js on line 30?

Reply:
I had troubles with the blur event when I tried to uplift to react-popper 1.x.x. To alleviate those concerns, I used the version that react-datepicker was using.

Reply:
Popper is removed though. https://github.com/cerner/terra-framework/pull/573/commits/671facc0b342a9226a59e6b6b302369411194eeb